user_docs/en/userGuide.t2t

```diff
@@ -391,7 +395,7 @@ However, by default, when the System caret moves, the review cursor follows alon
 This can be toggled on and off.
 
 Note that braille follows the [focus #SystemFocus] and [caret #SystemCaret] by default, rather than object navigation and text review.
-If you want it to follow object navigation and text review instead, you need to [configure braille to be tethered to #BrailleTether] review.
+If you want it to follow object navigation and text review instead, you need to [configure braille to be tethered to #BrailleTether] review or automatically to focus or review.

```

Comment:
I think this phrase is used in a few places, but I find it hard to read. Adding quotes around the option values would help. Overall I think it's better to just refer to the general concept and people can follow the link to get the specific details. Otherwise if the available values for this setting are changed many places in the user guide need to be updated.

Reply:
> 
> 
> I think this phrase is used in a few places, but I find it hard to read. Adding quotes around the option values would help. Overall I think it's better to just refer to the general concept and people can follow the link to get the specific details. Otherwise if the available values for this setting are changed many places in the user guide need to be updated.

Actually there are not that many places. I realized just now that there was a contradiction in the user guite. In the section "object navigation" we say that braille is tethered to "automatically" by default, whereas in the section "reviewing text" we say that by default braille is tethered to focus.

I have aligned both descriptions now, by default braille is tethered to automatically follow all the focuses and cursors.
Regarding the settings, I have removed the link strings because in my view we should focus here on readability. It is anyway not very trivial for beginners to understand how focuses and cursors work, which belongs to which system etc. I think if someone wants to jump to the settings, this can be done from the "contents" section ant the top of the page anyway.